test/index.js

```diff
@@ -1006,6 +1021,19 @@ describe('stream', function() {
         [], [1, 3, 2], [2, 8, 7, 6], [3, 5, 4]
       ]);
     });
+    it('#179 nested streams atomic update', function() {
```

Comment:
Not sure what I feel about issue numbers in the test description. But, we can keep it if you find it useful :smile:

Reply:
I don't have a preference, Removed.